pkg/provider.go

```diff
@@ -68,11 +68,12 @@ func Provider() *schema.Provider {
 			},
 		},
 		ResourcesMap: map[string]*schema.Resource{
-			"oasis_deployment":    resourceDeployment(),
-			"oasis_ipwhitelist":   resourceIPWhitelist(),
-			"oasis_certificate":   resourceCertificate(),
-			"oasis_backup_policy": resourceBackupPolicy(),
-			"oasis_project":       resourceProject(),
+			"oasis_deployment":      resourceDeployment(),
+			"oasis_ipwhitelist":     resourceIPWhitelist(),
+			"oasis_certificate":     resourceCertificate(),
+			"oasis_backup_policy":   resourceBackupPolicy(),
+			"oasis_project":         resourceProject(),
+			"oasis_example_dataset": resourceExampleDataset(),
```

Comment:
Shouldn't this be `oasis_example_dataset_installation` ?

Reply:
I think it should, yes.